Makefile

```diff
@@ -0,0 +1,24 @@
+.PHONY: cs-check cs-fix help lint lint-composer lint-yaml test
+
+default: help
+
+help:
+	@grep -E '^[a-zA-Z_-]+:.*?##.*$$' $(MAKEFILE_LIST) | sort | awk '{split($$0, a, ":"); printf "\033[36m%-30s\033[0m %-30s %s\n", a[1], a[2], a[3]}'
+
+cs-check: ## to show files that need to be fixed
+	php-cs-fixer fix --ansi --verbose --diff --dry-run
+
+cs-fix: ## to fix files that need to be fixed
+	php-cs-fixer fix --verbose
+
+lint: ## to lint php, yaml & composer
+	lint-composer lint-composer cs-check lint-yaml
```

Comment:
Shouldn't this be on the same line as ``lint:`` ?

Reply:
added make call before lint-composer :+1: (i reserved same line for help description)